src/index.js

```diff
@@ -22,26 +22,23 @@ const keyFilePath = path.resolve(__dirname, KEY_FILENAME);
 /*
  * Indented JSON.stringify() alias.
  */
-const stringify = (value) => JSON.stringify(value, null, '  ');
+const stringify = value => JSON.stringify(value, null, ' ');
```

Comment:
Why not two spaces for JSON? I feel like it should not different from JS code indentation